README.md

```diff
@@ -1,24 +1,73 @@
-# README
+# RAILS TEMPLATE 
+This repo is a template to be used for creating new projects.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## System dependencies                                                                                                                                  
+  - Ruby (see .ruby-version)
+  - Rails 7.0+
```

Comment:
We should also add info about chosen frontend stack, because it's not that obvious since rails v7

```suggestion
  - Rails 7.0+
  - Hotwire (Stimulus+Turbo)
```